components/stepper/themes/spectrum-two.css

```diff
@@ -17,21 +17,17 @@
 		--spectrum-stepper-border-color-default: var(--spectrum-gray-500);
 		--spectrum-stepper-border-color-hover: var(--spectrum-gray-600);
 		--spectrum-stepper-border-color-focus: var(--spectrum-gray-800);
-		--spectrum-stepper-border-color-focus-hover: var(--spectrum-gray-800);
+		--spectrum-stepper-border-color-focus-hover: var(--spectrum-gray-900);
 		--spectrum-stepper-border-color-keyboard-focus: var(--spectrum-gray-900);
```

Comment:
The s2 tokens specs also say gray-800 for keyboard focus, but I think [Tarun's PR](https://github.com/adobe/spectrum-css/pull/3594) should take care of that.